### ISSUE TITLE
Server error: TypeError: Cannot read property 'username' of undefined

### ISSUE THREAD

**Issue by clochix** [cozy/cozy-proxy#230]
Trace:
```
Cannot read property 'username' of undefined
TypeError: Cannot read property 'username' of undefined
    at /usr/local/cozy/apps/proxy/build/server/controllers/authentication.js:110:13
    at /usr/local/cozy/apps/proxy/build/server/controllers/authentication.js:26:14
    at /usr/local/cozy/apps/proxy/build/server/models/user.js:77:14
    at /usr/local/cozy/apps/proxy/build/server/models/user.js:64:14
    at /usr/local/cozy/apps/proxy/node_modules/americano-cozy/node_modules/jugglingdb-cozy-adapter/lib/cozy_data_system.js:403:18
    at parseBody (/usr/local/cozy/apps/proxy/node_modules/americano-cozy/node_modules/jugglingdb-cozy-adapter/node_modules/request-json/main.js:26:12)
    at Request._callback (/usr/local/cozy/apps/proxy/node_modules/americano-cozy/node_modules/jugglingdb-cozy-adapter/node_modules/request-json/main.js:81:18)
    at self.callback (/usr/local/cozy/apps/proxy/node_modules/americano-cozy/node_modules/jugglingdb-cozy-adapter/node_modules/request-json/node_modules/request/index.js:148:22)
    at Request.emit (events.js:95:17)
    at ClientRequest.self.clientErrorHandler (/usr/local/cozy/apps/proxy/node_modules/americano-cozy/node_modules/jugglingdb-cozy-adapter/node_modules/request-json/node_modules/request/index.js:257:10)

```

**Comment by clochix** [comment]
Already fixed in 18e2099 :-S

**Comment by poupotte** [comment]
Thanks @clochix
